docs/drash/v2.x/2_tutorials/6_services/5_drash_approved_services/etag.md

```diff
@@ -0,0 +1,137 @@
+# ETag
```

Comment:
Only problem with this is the service is still named `EtagService`, not `ETagService`, do we rename it in drash or rename it here?

Reply:
https://github.com/drashland/drash/commit/c2f6831ef9640919daca78b4e63a3a3b5e1373cc

Reply:
i changed it 😬